routes/users.js

```diff
@@ -2,30 +2,22 @@ var express = require("express");
 var router = express.Router();
 var db = require("../db.js");
 
-router.route("/")
+router
+  .route("/")
```

Comment:
I would leave the `.route()` at the top level, since the `.get()`, `.post()`, etc. are all sub-functions of the route function